unlock-app/src/components/content/CheckoutContent.tsx

```diff
@@ -43,6 +44,11 @@ export const CheckoutContent = ({
   configFromSearch,
   errors,
 }: CheckoutContentProps) => {
+  const { loading } = useProvider()
+  if (loading) {
+    return <></>
```

Comment:
Do we want to return nothing here? That will likely cause an increase in perceived latency for the end user

Reply:
That's a good point. I think the _pick provider_ logic will soon be moved to another place. I will address this in there.